board/safety/safety_toyota.h

```diff
@@ -87,14 +98,31 @@ static void toyota_rx_hook(const CANPacket_t *to_push) {
     }
 
     // enter controls on rising edge of ACC, exit controls on ACC off
-    // exit controls on rising edge of gas press
-    if (addr == 0x1D2) {
-      // 5th bit is CRUISE_ACTIVE
```

Comment:
keep these comments?

Reply:
The "rising edge of gas press" comment was still there. I added signal annotations for each signal inside the block.

The same brake-pressed naming was used for three different messages, so I had to annotate the DBC as well there.